blocks/columns/columns.css

```diff
@@ -23,6 +23,7 @@
 
 .columns .columns-img-col {
   position: relative;
+  display: flex;
```

Comment:
@seanmilfort, I think this change unintentionally impacts some other layouts on other pages that use this class.

Compare the layout of green circle images/icons on the Members / Mail Order page

Before: https://main--elixirsolutions--hlxsites.hlx.live/members/about-mail  (green images are stacked on top of the text)

After: https://184-remove-bubbles--elixirsolutions--hlxsites.hlx.live/members/about-mail  (green images are aligned to the left of the text)

I'm not sure but you might be able to use
`align-self: center;`
instead of 
`display: flex;`